service/controller/v10/resource/s3bucket/resource.go

```diff
@@ -127,3 +130,13 @@ func (r *Resource) getS3BucketTags(customObject v1alpha1.AWSConfig) []*s3.Tag {
 
 	return s3Tags
 }
+
+func (r *Resource) canBeDeleted(bucket BucketState) bool {
+	if bucket.IsLoggingBucket {
```

Comment:
This function could be just `return bucket.IsLoggingBucket && !r.TestingEnvironment`.

Reply:
I think this is a nice simplification. Would be good to implement if its straight forward.

Reply:
indeed v10 should be untouched because is frozen. In the v11 is done as suggested. Let me remove changes from v10 :)

Reply:
Good catch. Thanks for removing.